packages/frontend/src/redux/slices/multiplier/index.js

```diff
@@ -0,0 +1,64 @@
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
+import { wallet } from "../../../utils/wallet";
+import { createSelector } from "reselect";
+import { ACCOUNT_ID_SUFFIX } from "../../../config"
+
+
+
+const SLICE_NAME = "multiplier";
+
+const initialState = {
+    prices: {},
+};
+
+export const fetchMultiplier = createAsyncThunk(
+    `${SLICE_NAME}/fetchMultiplier`,
+    async function () {
+        try {
+            const response = await wallet.connection.provider.sendJsonRpc(
+                "query",
+                {
+                    request_type: "call_function",
+                    account_id: `priceoracle.${ACCOUNT_ID_SUFFIX}`,
+                    method_name: "get_price_data",
+                    args_base64: "eyJhc3NldF9pZHMiOiBbIndyYXAudGVzdG5ldCJdfQ==",
```

Comment:
can we pass in `asset_id`s in a human readable way so we can iterate on this quicker if we need to?